docs/content/docs/dev/table/sqlClient.md

```diff
@@ -69,6 +69,12 @@ In the gateway mode, the CLI submits the SQL to the specified remote gateway to
 
 The `<gateway address>` can be provided in two formats: as a `host:port` combination or as a full URL.
 
+You can also override the default job manager configuration in SQL client by explicitly providing the cluster address while initialising the client as below:
+
+```bash
+./bin/sql-client.sh gateway --endpoint <gateway address> -Drest.address=<flink cluster host> -Drest.port=<port>
```

Comment:
This does not look very consistent. 
- endpoint <gateway address>. - I assume this will include the port so it should be <gateway endpoint>
- then the other endpoint for the JM is defined as an address and a port.

Is this an opportunity to make them more consistent?

Reply:
I suggest ,without changing the code we change `gateway address` to `gateway endpoint`. As address implies IP address and does not include the port.